tests/framework/extensions/clusters/clusters.go

```diff
@@ -625,16 +626,29 @@ func AgentAffinityConfigHelper(advancedClusterAffinity *management.Affinity) *co
 func HardenK3SRKE2ClusterConfig(clusterName, namespace string, clustersConfig *ClusterConfig, machinePools []apisV1.RKEMachinePool, cloudCredentialSecretName string) *apisV1.Cluster {
 	v1Cluster := NewK3SRKE2ClusterConfig(clusterName, namespace, clustersConfig, machinePools, cloudCredentialSecretName)
 
-	if strings.Contains(v1Cluster.Spec.KubernetesVersion, "k3s") {
-		v1Cluster.Spec.RKEConfig.MachineGlobalConfig.Data["kube-apiserver-arg"] = []string{
-			"enable-admission-plugins=NodeRestriction,PodSecurityPolicy,ServiceAccount",
-			"audit-policy-file=/var/lib/rancher/k3s/server/audit.yaml",
-			"audit-log-path=/var/lib/rancher/k3s/server/logs/audit.log",
-			"audit-log-maxage=30",
-			"audit-log-maxbackup=10",
-			"audit-log-maxsize=100",
-			"request-timeout=300s",
-			"service-account-lookup=true",
+	if strings.Contains(clustersConfig.KubernetesVersion, "k3s") {
+		if clustersConfig.KubernetesVersion <= HardenedKubeVersion {
```

Comment:
nit; could we update the name of this const now that we will harden any version? maybe `PSPKubeVersionLimit` or something

Reply:
Yeah, I can update the name